x-pack/plugins/uptime/public/components/fleet_package/browser/formatters.ts

```diff
@@ -28,7 +28,8 @@ export const browserFormatters: BrowserFormatMap = {
   [ConfigKeys.SOURCE_ZIP_PASSWORD]: null,
   [ConfigKeys.SOURCE_ZIP_FOLDER]: null,
   [ConfigKeys.SOURCE_ZIP_PROXY_URL]: null,
-  [ConfigKeys.SOURCE_INLINE]: (fields) => stringToJsonFormatter(fields[ConfigKeys.SOURCE_INLINE]),
+  [ConfigKeys.SOURCE_INLINE]: (fields) => fields[ConfigKeys.SOURCE_INLINE],
+  // [ConfigKeys.SOURCE_INLINE]: (fields) => stringToJsonFormatter(fields[ConfigKeys.SOURCE_INLINE]),
```

Comment:
@shahzad31 looks like we'll need to copy formatters and have divergent formatters for both views.